### DISCUSSION TITLE
about spi.  orangepi 3b

### DISCUSSION THREAD

**Discussion by mysteryzzp** [Joshua-Riek/ubuntu-rockchip#1055] [General]
i flash ubuntu 24.04 on sd card.and it boot successfully.
but the nvme disk doesn't display.
so i execute 'u-boot-install-mtd'  and 'u-boot-update'.
and then the system cannot boot.
the only red led lights.but the green led doesn't.

so how should i recovery it?

and what does u-boot-install-mtd and u_boot-xxxx   do ?

**Comment by FreeWilly2022** [comment]
You probably need to connect to the console using a uart to USB converter, instructions are on page 122 of the Orange Pi 3B User Manual, available on the orange pi website.
